src/components/NewsletterComponents/Columns.tsx

```diff
@@ -116,7 +116,7 @@ export const columns: ColumnDef<Newsletter>[] = [
     accessorKey: "actions",
     header: () => <div className="text-right pr-4">Actions</div>,
     cell: ({ row }) => {
-      const [, setData] = useRecoilState(newsletterData);
+      const [, setData] = useRecoilState(campaignData);
```

Comment:
This can be changed to `useSetRecoilState`. Look into the different functionality of Recoil

Reply:
Replaced useRecoilState with useSetRecoilState in the latest commit